filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java

```diff
@@ -101,10 +100,10 @@ public void testXmlToSqlSortBy() throws JAXBException {
         xml = TestFes2_0_2.class.getResourceAsStream("filter_Sorting.xml");
         Object elementObject = "filter_Sorting";
 
-        XmlToSql(elementObject).toString();
+        XmlToSql(elementObject);
 
         //Error : objectFromFilterXml is null
-        XmlToSql(null).toString();
+        XmlToSql(null);
     }
```

Comment:
For those three cases, I need to test if the message of the logger is launch. It's why I don't use an assert. But I don't know if there is an other way.